release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -2151,6 +2151,13 @@ To avoid this issue in 4.4, do not spin up multiple Pods with a Guaranteed QoS o
 If you encounter this issue, delete and then recreate the Pod.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1834979[*BZ#1834979*])
 
+* VMs with the `runStrategy` attribute set to `Manual` do not indicate whether
+they should be running or stopped, and the web console incorrectly assumes they
```

Comment:
> ! should v
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as
appropriate.

I'd rewrite to avoid "should" if possible and consider moving the restriction to the start.

Reply:
Changed "should" to "are" because I think that still works to address the bug. The restriction is a workaround, so I htink I would leave it where it is to match the others but perhaps preface it with "To avoid this issue, ...."